README.md

```diff
@@ -116,11 +117,17 @@ assets: {
 }   
 ```
 
+#### options.createCopies
+Type: `Boolean`
+Default value: `true`
+
+When set to `false`, `cachebust` will not create hashed copies of the files. Useful if you use server rewrites to serve your files.
+
 #### options.deleteOriginals
 Type: `Boolean`  
 Default value: `false`
 
-When set, `cachebust` will delete the original versions of the files that have been renamed.  For example, `style.css` will be deleted after being copied to `style.dcf1d324cb50a1f9.css`.
+When set, `cachebust` will delete the original versions of the files that have been copied. For example, `style.css` will be deleted after being copied to `style.dcf1d324cb50a1f9.css`. Will not delete original files with `options.createCopies` set to `false`.
```

Comment:
Can we change the word `copied` to `hashed` - that terminology is used in other places in this README. Also, I wouldn't have the addition of that last sentence - see other comment for reasoning